flow360/component/volume_mesh.py

```diff
@@ -397,6 +299,14 @@ def __init__(
                 )
             self.params = params.copy(deep=True)
 
+        if file_name is not None:
+            mesh_parser = MeshNameParser(file_name)
+            if not mesh_parser.is_valid_volume_mesh():
+                raise Flow360ValueError(
+                    f"Unsupported volume mesh file extensions: {mesh_parser.format.ext()}. "
+                    f"Supported: [{MeshFileFormat.STL.ext()},{MeshFileFormat.UGRID.ext()},{MeshFileFormat.CGNS.ext()}]."
```

Comment:
I think "STL" is not supported volume mesh, right?

Reply:
fixed